SmartSyncExplorerReactNative/js/StoreMgr.js

```diff
@@ -25,7 +25,14 @@
  */
 
 import EventEmitter from './events';
-import {smartstore, smartsync} from 'react-native-force';
+import {smartstore, smartsync, forceUtil} from 'react-native-force';
+
+const registerSoup = forceUtil.promiser(smartstore.registerSoup);
```

Comment:
Using promise-based version of the APIs
For sync operations using promiserNoRejection which funnels success and error to the `then()`